FF1Lib/FF1Rom.cs

```diff
@@ -114,7 +115,10 @@ public void Randomize(Blob seed, Flags flags, Preferences preferences)
 				rng = new MT19337(BitConverter.ToUInt32(hash, 0));
 			}
 			if (flags.TournamentSafe) AssureSafe();
-
+			// Create a SHA-1 String of the original input rom (used in Rom Stamping process later)
+			using (SHA1 hasher = SHA1.Create()){
```

Comment:
Can you not do this inline? Also the comment isn't really necessary since it's very clear what is being done here.

`inputRomHash = SHA1.Create().ComputeHash(Data.ToBytes())